src/commands/trace/interfaces.ts

```diff
@@ -0,0 +1,21 @@
+import {AxiosPromise, AxiosResponse} from 'axios'
+
+export interface CustomIDs {
+  id: string
+  parent_id?: string
+}
+
+export interface Payload {
+  command: string
+  custom: CustomIDs
+  data: Record<string, string>
```

Comment:
this data are the env vars dictionary, isn't it? Could we update the name so that it reflects that? If the key is already defined in the backend, either: 
* have interfaces with different (more meaningful) keys here and map them at the end, when doing `request` or
* add comments

Reply:
was this addressed?

Reply:
it has, now! 🙈